samples/dialogs/src/main/java/com/badoo/ribs/samples/dialogs/rib/dummy/DummyPresenter.kt

```diff
@@ -0,0 +1,19 @@
+package com.badoo.ribs.samples.dialogs.rib.dummy
+
+import com.badoo.ribs.core.plugin.RibAware
+
+interface DummyPresenter {
+
+    fun handle(event: DummyView.Event)
+}
+
+internal class DummyPresenterImpl(
+    private val ribAware: RibAware<Dummy>
```

Comment:
You could move the default argument here from the Builder:

```suggestion
    private val ribAware: RibAware<Dummy> = RibAwareImpl()
```